### ISSUE TITLE
OverlayScrollbars.js:5102 Uncaught TypeError: Right-hand side of 'instanceof' is not an object

### ISSUE THREAD

**Issue by mbrainiac** [KingSora/OverlayScrollbars#25]
Hi,
I got this error when use scroll method, OverlayScrollbars without jQuery.

```
var possibleElementIsJQuery = possibleElement instanceof helper || possibleElement instanceof window.jQuery;
```

OverlayScrollbars.js:5102 Uncaught TypeError: Right-hand side of 'instanceof' is not an object
    at window.(anonymous function).OverlayScrollbarsInstance._base.scroll (webpack:///./node_modules/overlayscrollbars/js/OverlayScrollbars.js?:5102:104)

**Comment by KingSora** [comment]
Thank you, I'll fix this as soon as possible!

**Comment by KingSora** [comment]
The bug has been fixed in version [1.4.3](https://github.com/KingSora/OverlayScrollbars/releases/tag/v1.4.3).
Once again, thank you for reporting!
